cegs_portal/search/view_models/v1/dna_features.py

```diff
@@ -274,7 +274,7 @@ def loc_search(
 
         included_fcp = feature_properties_set & FUNCTIONAL_CHARACTERIZATION_PROPERTIES
 
-        if LocSearchProperty.REG_EFFECTS in feature_properties or len(included_fcp) > 0:
+        if LocSearchProperty.REG_EFFECTS in feature_properties:
```

Comment:
This also shaves a lot of time -- this stuff isn't needed for the functional characterization properties